### PR TITLE
Make emscripten build work with/for modern versions. Cleanup.

### PR DESCRIPTION
**Pull request by adriweb**
Tested locally with emscripten 3.1.50, works fine.

Updated the demo hosted here https://tiplanet.org/scripts/luna-web/index.html

### CODE REVIEW

**Comment by Vogtinator** [comment]
Is there no replacement for FileSaver meanwhile?

**Comment by adriweb** [comment]
Hmm I suppose [this](https://stackoverflow.com/questions/2897619/using-html5-javascript-to-generate-and-save-a-file/71744407#71744407) works everywhere now... we should probably use that and drop the external dep altogether.

**Comment by adriweb** [comment]
Removed FileSaver.js

File: emscripten/Makefile [hunk 1]
```diff
@@ -1,29 +1,28 @@
 CC      = emcc
 
-CFLAGS  = -O3 -W -Wall -Wextra -DNOCRYPT -DUSE_FILE32API -Wno-unused-parameter
-CFLAGS += --llvm-lto 3 -s USE_ZLIB=1 -s INVOKE_RUN=0 -s NO_EXIT_RUNTIME=1 --pre-js luna-utils.js
+EMFLAGS := --memory-init-file 0 -s WASM=1 -s MODULARIZE=1 -s EXPORT_ES6=1 -s EXPORT_NAME="'WebLuna'" -s INVOKE_RUN=0 -s NO_EXIT_RUNTIME=1 -s ASSERTIONS=0 -s DISABLE_EXCEPTION_CATCHING=1 -s EXPORTED_RUNTIME_METHODS="['FS','callMain']"
```
Comment (Vogtinator) [inline emscripten/Makefile:3]:
```suggestion
EMFLAGS := -s WASM=1 -s MODULARIZE=1 -s EXPORT_ES6=1 -s EXPORT_NAME="'WebLuna'" -s INVOKE_RUN=0 -s NO_EXIT_RUNTIME=1 -s EXPORTED_RUNTIME_METHODS="['FS','callMain']"
```

Comment (adriweb) [reply emscripten/Makefile]: done

**Pull request merged**